tools/lint/tests/test_file_lints.py

```diff
@@ -809,6 +809,108 @@ def test_invalid_meta_file():
     assert errors == []
 
 
+@pytest.mark.parametrize("files,yml,expected_errors", [
+    (
+        ["file1.txt", "file2.txt", "file3.txt"],
+        b"""\
+features:
+- name: feature1
+  files:
+  - file1.txt
+""",
+        []
+    ),
+    (
+        ["file1.txt", "file2.txt", "file3.txt"],
+        b"""\
+features:
+- name: feature1
+  files:
+  - file*.txt
+""",
+        []
+    ),
+    (
+        ["file1.txt", "file2.txt", "file3.txt"],
+        b"""\
+features:
+- name: feature1
+  files:
+  - file*.txt
```

Comment:
Can you also add a test with a pattern that doesn't match? Ideally one pattern that matches something and another that matches something in the same test, so that we know that the check is at the level of the pattern, not that all files/patterns together matches something.

Reply:
Added! Will merge after clean CI run.